smartrack-manifest.xml

```diff
@@ -9,26 +9,41 @@
   <remote alias="repo" fetch="https://github.com/rauc" name="rauc"/>
   
   <default sync-j="4"/>
+  <!-- bitbake: Branch 1.46 -->
```

Comment:
I should have post my question (about the hash on "revision" field) as a review comment instead of a general comment, so that we can reply to it directly..

Yea I'm okay if it is pointing to an arbitrary comment instead of a specific tag. I don't think it will break thing as long as we are following Toradex manifest (I trust them has run some level of test). My thought was if it pointing to a specific tag, then we can list the tag here as well, e.g. <!-- bitbake: Branch 1.46    Tag 1.46_Stable -->, so that we can easily know if we are pulling a branch that's still under development or it's already stable.

Anyways, before we conduct final V&V or the branch has become stable (see whoever come first), we need to update all these hash to point to the "latest stable" commit, as pointing to a "under development" branch is error prone